app/controllers/posts_controller.rb

```diff
@@ -3,6 +3,14 @@ class PostsController < ApplicationController
   skip_authorization_check only: [:create]
   def index
```

Comment:
- [ ] When I try to test your **_API endpoint_** using an **_API tester_**, I get this error: 

![image](https://github.com/ismayito/Rails-blog-app/assets/77106962/c1f00d85-2c0c-4444-814c-bafa0f95fbb4)

---
Instead of skipping the authorization check, you can do the following:

```ruby
  skip_before_action :verify_authenticity_token, only: %i[create]
```

This will prevent rails from authenticating the CRSF token for this endpoint. 👍 💻